README.md

```diff
@@ -126,6 +123,10 @@ We are totally Hacktoberfest friendly, in 2020 we are even giving Switchblade St
 <a href="https://opencollective.com/switchblade/sponsor/9/website?requireActive=false" target="_blank"><img src="https://opencollective.com/switchblade/sponsor/9/avatar.svg?requireActive=false"></a>
 <a href="https://opencollective.com/switchblade/sponsor/10/website?requireActive=false" target="_blank"><img src="https://opencollective.com/switchblade/sponsor/10/avatar.svg?requireActive=false"></a>
 
+<h2 align="center">Hacktoberfest</h2>
+
+We are totally Hacktoberfest friendly, in 2020 we are even giving Switchblade Stickers to those who open 3 or more _valid_ PRs to Switchblade during October, please check our issue tab clicking [here.][hacktoberfest-url]
+
```

Comment:
Move it to below "Backers" please :)

Reply:
Done, "Hacktoberfest" section is now below "Backers"